MAUI/ListView/item-drag-and-drop.md

```diff
@@ -0,0 +1,422 @@
+
+{% tabs %}
+{% highlight xaml hl_lines="15"%}
+<ContentPage xmlns:syncfusion="clr-namespace:Syncfusion.Maui.ListView;assembly=Syncfusion.Maui.ListView">
+  <syncfusion:SfListView x:Name="listView" 
+                   ItemsSource="{Binding ToDoList}"
+                   DragStartMode="OnDragIndicator"
+                   BackgroundColor="#FFE8E8EC"
+                   ItemSize="60">
+  <syncfusion:SfListView.ItemTemplate>
+    <DataTemplate>
+      <Grid Padding="10">
+          <Grid.ColumnDefinitions>
+            <ColumnDefinition Width="*" />
+            <ColumnDefinition Width="60" />
+          </Grid.ColumnDefinitions>
+          <Label x:Name="textLabel" Text="{Binding Name}" Grid.Column="1" FontSize="15" TextColor="#333333" />
+          <syncfusion:DragIndicatorView Grid.Column="2" ListView="{x:Reference listView}" 
+                    HorizontalOptions="Center" 
+                    VerticalOptions="Center">
+            <Grid Padding="10, 20, 20, 20">
+              <Image Source="dragindicator.png" />
+            </Grid>
+          </syncfusion:DragIndicatorView>
+        </Grid>
+      </Grid>
+    </DataTemplate>
+  </syncfusion:SfListView.ItemTemplate>
+</syncfusion:SfListView>                
+</ContentPage>
+{% endhighlight %}
+{% highlight c# hl_lines="17"%}
+listView.ItemTemplate = new DataTemplate(() =>
+{
+  var grid = new Grid();
+  var name = new Label
+  {
+    FontSize = 15,
+    VerticalOptions = LayoutOptions.Center
+  };
+  name.SetBinding(Label.TextProperty, new Binding("Name"));
+
+  var indicatorGrid = new Grid()
+  {
+    Padding = new Thickness(10, 20, 20, 20),
+    HorizontalOptions = LayoutOptions.End,
+    VerticalOptions = LayoutOptions.Center
+  };
+  var dragIndicatorView = new DragIndicatorView() { ListView = this.listView };
+  var indicator = new Image() { Source = "dragindicator.png" };
+
+  indicatorGrid.Children.Add(indicator);
+  dragIndicatorView.Content = indicatorGrid;
+
+  grid.Children.Add(name);
+  grid.Children.Add(dragIndicatorView);
+  grid.SetRow(dragIndicatorView, 1);
+  grid.SetColumn(dragIndicatorView, 0);
+  return grid;
+});
+{% endhighlight %}
+{% endtabs %}
+
+The screenshot shows the output of the reordering items by drag and drop.Download the entire source code from GitHub [here](https://github.com/SyncfusionExamples/how-to-load-drag-view-indicator-in-.net-maui-listview.).
+
+![Item reordering by drag and drop](Images/item-drag-and-drop/maui-listview-dragviewindicator.gif)
+
+## Drag item customization
+
+By defining the `SfListView.DragItemTemplate` property of the `SfListView`, you can display a custom user interface (UI) when performing drag and drop operations. The template can be defined either in code or XAML.
+
+N> If the `BackgroundColor` is set for the `DragItemTemplate` or `DragIndicatorView`, make sure to set `InputTransparent` to `true`. This is necessary because dragging does not occur when using the `DragIndicatorView` in UWP.
```

Comment:
Please ensure this case in WinUI

Reply:
Addressed the concerns provided. Tested   DragIndicatorView with BackgroundColor,dragging was working fine.So removed these lines.